src/Numerics/project.json

```diff
@@ -0,0 +1,97 @@
+﻿{
```

Comment:
Have used this approach on some other projects and haven't had any issues yet. I think this will work OK until we can migrate to VS2017 after RTM and merge everything into a unified csproj file.